.github/workflows/main_branch.yml

```diff
@@ -1,4 +1,4 @@
-name: Main Branch Workflow
+name: Scheduled benchmarks
```

Comment:
Thanks, it was a hasty change. I think word "scheduled" fits well, but "benchmarks" is a mistake.

I would not use term "cross-platform" for this job because the commit-based job is cross-platform, too.

Reply:
Let's call it "weekly" testing.